app/profile/utils/get-pathname.ts

```diff
@@ -0,0 +1,11 @@
+export function GetPathname(pathname: string) {
+  const paths = pathname.split('/')
+
+  let routeName = ''
+
+  if (paths.length > 3) {
+    return (routeName = `-${paths[4]}`)
+  }
+
+  return routeName
```

Comment:
```suggestion
  const paths = pathname.split('/')
  
  if (paths.length > 3) 
    return `-${paths[4]}`

  return ''
```

Or maybe empty routeName should be home or overview?